CONTRIBUTING.md

```diff
@@ -96,6 +96,7 @@ and follow the template below:
   title: Category Name
   icon: icon-class
 ```
+The `icon-class` value needs to be chosen from [Semantic-Ui][semantic-ui]https://semantic-ui.com/elements/icon.html.
```

Comment:
Why do you display it twice?

Reply:
I meant to remove the link after adding it in the footer.  Great catch.  I have removed the accidental duplication.